Project.toml

```diff
@@ -23,18 +24,19 @@ AdvancedMHMCMCChainsExt = "MCMCChains"
 AdvancedMHStructArraysExt = "StructArrays"
 
 [compat]
-AbstractMCMC = "5"
+AbstractMCMC = "5.6"
+BangBang = "0.3.19, 0.4"
```

Comment:
I'm assuming this is left-over from when you were using BangBang.jl? We're no longer using it, right?